integrations/huggingface_hub/scripts/push_to_hub.py

```diff
@@ -0,0 +1,106 @@
+import os
+import typer
+import tempfile
+import tarfile
+import shutil
+import json
+from huggingface_hub import Repository, HfApi, HfFolder
+
+token_classification_components = ["ner", "tagger", "morphologizer"]
+text_classification_components = ["textcat", "textcat_multilabel"]
+
+
+def list_tags(name, tags):
+    if tags is None:
+        return ""
+    if isinstance(tags, str):
+        tags = [tags]
+    if len(tags) == 0:
+        return ""
+    name_tags = [f"- {tag}" for tag in tags]
+    return f"{name}:\n" + "\n".join(name_tags) + "\n"
```

Comment:
in `transformers` we switched to generating this via `pyyaml` which adds a dependency but is probably less error prone in the long term

Reply:
Changed!